packages/shorebird_cli/lib/src/commands/release/release_android_command.dart

```diff
@@ -172,16 +188,27 @@ ${summary.join('\n')}
       flavor: flavor,
     );
 
-    logger
-      ..success('\n✅ Published Release!')
-      ..info('''
+    final instructions = generateApk
+        ? '''
+
+Your next step is to upload the apk to the Play Store.
+${lightCyan.wrap(apkPath)}
+
+See the following link for more information:    
+${link(uri: Uri.parse('https://support.google.com/googleplay/android-developer/answer/9859152?hl=en'))}
+'''
+        : '''
 
 Your next step is to upload the app bundle to the Play Store.
 ${lightCyan.wrap(bundlePath)}
 
 See the following link for more information:    
 ${link(uri: Uri.parse('https://support.google.com/googleplay/android-developer/answer/9859152?hl=en'))}
```

Comment:
It might be nice to have this be a single string and provide "app bundle"/"apk" as variables, as that's the only difference between the two.